File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
dice1, dice2, dice3 = map(int, input().split())

if(dice1 == dice2 and dice2 == dice3 and dice3 == dice1):
    print(10000 + dice1 * 1000)
elif(dice1 == dice2 or dice2 == dice3 or dice3 == dice1):
    if(dice1 == dice2 or dice3 == dice1):
        print(1000 + dice1 * 100)
    else:
        print(1000 + dice2 * 100)
else:
    max = dice1
    if(dice2 > max):
        max = dice2
    if(dice3 > max):
        max = dice3
    print(max * 100)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int dice1 = sc.nextInt();
		int dice2 = sc.nextInt();
		int dice3 = sc.nextInt();
		sc.close();

		if (dice1 == dice2 && dice2 == dice3 && dice3 == dice1) {
			System.out.println(10000 + dice1 * 1000);
		} else if (dice1 == dice2 || dice2 == dice3 || dice3 == dice1) {
			if (dice1 == dice2 || dice3 == dice1) {
				System.out.println(1000 + dice1 * 100);
			} else {
				System.out.println(1000 + dice2 * 100);
			}
		} else {
			int max = dice1;
			if (dice2 > max) {
				max = dice2;
			}
			if (dice3 > max) {
				max = dice3;
			}
			System.out.println(max * 100);
		}
	}
}